guides/source/association_basics.md

```diff
@@ -2350,6 +2350,17 @@ end
 
 If a `before_add` callback throws an exception, the object does not get added to the collection. Similarly, if a `before_remove` callback throws an exception, the object does not get removed from the collection.
 
+Note that these callbacks are called only when the associated objects are added or removed through the association collection:
```

Comment:
NOTE: These callbacks are called only when the associated objects are added or removed through the association collection:
```suggestion
Note that these callbacks are called only when the associated objects are added or removed through the association collection:
```